tests/symbols/spec.svg

```diff
@@ -0,0 +1,16 @@
+<svg id="svgElement" width="500" height="500" xmlns="http://www.w3.org/2000/svg">
+  <symbol id="myDot" x="1" height="100" viewBox="0 0 80 100" >                                          
+    <circle cx="50" cy="50" r="50"/>
+  </symbol>
+  <symbol id="myDot-overflow" height="100" overflow="visible" viewBox="0 0 80 100" >                                          
+    <circle cx="50" cy="50" r="50"/>
+  </symbol>
+  
+  <use href="#myDot"   />    
```

Comment:
maybe you could align the circles a little better and make them distinguishable, so it's immediately clear which circle is which.